from src.utils import return_random_number

try:
  return_random_number(0,200, 45):
  print("The reurtn_random_number function works")
except:
  print("We have an issue with return_random_number function")
